iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -1030,6 +1039,48 @@ static void copyFromAliasingBufferToResultBuffer(OpBuilder &b, Location loc,
   }
 }
 
+static LogicalResult convertPadTensorOp(
+    OpBuilder &b, linalg::PadTensorOp padTensorOp, BlockAndValueMapping &bvm,
+    BufferizationPlan &plan, WorkgroupMemoryAllocationFn allocationFn) {
+  auto inputTensor = padTensorOp.source();
+  auto inputMemref = bvm.lookup(inputTensor);
+
+  auto loc = padTensorOp.getLoc();
+
+  if (failed(getOrAllocateResultBuffers(b, padTensorOp, padTensorOp.result(),
```

Comment:
Move this into the `TypeSwitch` statement below (to be consistent with how everything else is done). Ideally these patterns should have to deal with `plan` etc. etc. They rely on the result tensor already being mapped to the correct buffer. Basically this function shouldnt need `plan` or `allocationFn`.